experiments/compositions/composition/config/expanders/expander_versions.yaml

```diff
@@ -15,11 +15,12 @@
 apiVersion: composition.google.com/v1alpha1
 kind: ExpanderVersion
 metadata:
-  name: jinja2
+  name: pjinja2
```

Comment:
*p*jinja2 ?

Reply:
Ahh. I think I see. Should we make this more explicit? Jinja2-job ? The following then might be the jinja2-service, jinja2-grpc?

Reply:
It only matters if we are doing the same expander type in both modes. Jinja is an exception since we started as a job and are moving towards grpc. ideally we should drop the job,. but keeping it around for just-in-case.

 So will rename: pjinja2 -> jinja2-job 
But will not rename: jinja2 -> jinja2-grpc X